### ISSUE TITLE
Can’t adjust lock screen components page?

### ISSUE THREAD

**Issue by gleep52** [midnightchip/Asteroid#93]
I erased and reset my phone, restored from iCloud, jailbroke with unc0ver 3.2.1 and installed rocketbootstrap and cephei... Asteroid works, and I can adjust all the settings for asteroid, except the components link on the lock screen page.

**Comment by Salkio** [comment]
I have the same issue. Also restored with rootFS and tried to reset all contents and settings (after doing rootFS). Cannot access contents page !

**Comment by Salkio** [comment]
Hey try installing the tweak called shuffle after that this fixed the issue 😂

**Comment by gleep52** [comment]
Yes, it's not the RIGHT way, but it DOES fix the glitch... I wonder why?  Many thanks!

**Comment by Salkio** [comment]
It fixed it for me yeah. But it's a cook tweak tho. Organises tweaks in one category.

**Comment by blk380** [comment]
I'm having the same issue, but installing shuffle fixed it as well..

**Comment by maxime6969** [comment]
Had the same issue, instelling shuffle fixed iT all.

**Comment by JeromeAstero** [comment]
Yeah! I have this issue too.
iPhone 6s n71map 12.1.2 unc0ver

**Comment by the-casle** [comment]
Yea, I'll try to talk to the dev of shuffle. Really confused but its an issue with libscspreferences and not actually asteroid

**Comment by b4r0z** [comment]
same here ip7 and ip7+ 12.4
